src/zoo/owl_zoo_cmd.ml

```diff
@@ -35,10 +48,11 @@ let remove_gist gist =
   Sys.command cmd |> ignore
 
 
-let upload_gist gist =
-  Owl_log.debug "owl_zoo: %s uploading" gist;
-  let cmd = Printf.sprintf "owl_upload_gist.sh %s" gist in
-  Sys.command cmd |> ignore
+let upload_gist dir =
+  Owl_log.debug "owl_zoo: %s uploading" dir;
+  let cmd = Printf.sprintf "owl_upload_gist.sh %s" dir in
+  Sys.command cmd |> ignore;
+  _read_file (dir ^ "/gist.id") |> _strip_last_char
```

Comment:
What is this for? Is this for removing the `\n` in the string? If so, this is not the right way because what if the last char is not `\n`?

Reply:
Why don't you just use the `read_file` function in `Owl_utils` which already contains a `trim` parameter?